docs/api-reference.md

```
@@ -0,0 +1,256 @@
+    "pageSize": 20,
+    "totalPages": 5,
+    "totalItems": 98
+  }
+}
+```
+
+## Filtros
+
+Suporte a filtros via query string:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)